### PR TITLE
docs(README): Update README

### PR DESCRIPTION
**Pull request by emrcftci**
- Add link to ExampleApp directory and GitHub issues

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ To improve build times, and your productivity in this situation, why not swap in
 
 ## Installation
 
-`[sudo] gem install xcode_barber`
+`sudo gem install xcode_barber`
```
Comment (michaeleisel) [inline README.md]: sudo is not always required, it depends on the ruby setup

**Comment by michaeleisel** [comment]
one thing

File: README.md (hunk 1 above)
Comment (emrcftci) [reply README.md]: Ooops its my mistake, thanks for the comment.
Done ✅

**Comment by michaeleisel** [comment]
Thanks!

**Pull request merged**